client/components/disputed-order-notice/index.js

```diff
@@ -108,6 +108,90 @@ const DisputedOrderNoticeHandler = ( { chargeId, onDisableOrderRefund } ) => {
 	);
 };
 
+const UrgentDisputeNoticeBody = ( {
+	isPreDisputeInquiry,
+	disputeReason,
+	formattedAmount,
+	dueBy,
+	countdownDays,
+} ) => {
+	const formatString = isPreDisputeInquiry
+		? __(
+				// Translators: %1$s is the formatted dispute amount, %2$s is the dispute reason, %3$s is the due date.
+				"Please resolve the inquiry on this order for %1$s labeled '%2$s' by %3$s.",
+				'woocommerce-payments'
+		  )
+		: __(
+				// Translators: %1$s is the formatted dispute amount, %2$s is the dispute reason, %3$s is the due date.
+				"Please resolve the dispute on this order for %1$s labeled '%2$s' by %3$s.",
```

Comment:
```suggestion
				"Please resolve the dispute on this order of %1$s labeled '%2$s' by %3$s.",
```

Same as the above.

Reply:
Going with consistent `of` here, also less characters so slightly more lean to read.